crates/ruff_python_ast/src/nodes.rs

```diff
@@ -1500,6 +1507,9 @@ bitflags! {
         /// The string has an `r` or `R` prefix, meaning it is a raw string.
         /// It is invalid to set this flag if `U_PREFIX` is also set.
         const R_PREFIX = 1 << 3;
+
+        /// The string is invalid.
```

Comment:
To avoid this sounding strangely judgemental:

```suggestion
        /// The string was deemed invalid by the parser.
```